integration/integration_test.go

```diff
@@ -756,27 +780,12 @@ type gitServerOptions struct {
 func createGitServer(t *testing.T, opts gitServerOptions) string {
 	t.Helper()
 	if opts.authMW == nil {
-		opts.authMW = checkBasicAuth(opts.username, opts.password)
+		opts.authMW = gittest.BasicAuthMW(opts.username, opts.password)
 	}
 	srv := httptest.NewServer(opts.authMW(createGitHandler(t, opts)))
 	return srv.URL
 }
 
-func checkBasicAuth(username, password string) func(http.Handler) http.Handler {
```

Comment:
review: moved to `gittest`

Reply:
~~is it intended to be used anywhere else in envbuilder? I'm considering moving it to `integration/gittest.go`~~

Ok, I found the place. Speaking of consistency with coder/coder that should land in `testutil`.

Reply:
Will do in a follow-up PR 👍